### DISCUSSION TITLE
Git Blame: Select all lines from same commit when clicking on a commit

### DISCUSSION THREAD

**Discussion by jansol** [zed-industries/zed#23928] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Clicking on a commit in the git blame view acts the same way as clicking on a line number in the gutter. It could be useful to instead select all lines that have last been touched by that commit? Or perhaps even all lines that were touched by that commit, even if there have been other changes afterwards (not sure how complicated that would be to trace correctly).

**Comment by CurbaiCode** [comment]
I think combining duplicate consecutive blame lines into a single one would complement this feature.

**Comment by maxdeviant** [comment]
This is an interesting idea. I'm going to move it to a discussion for further evaluation.
